paddlenlp/transformers/squeezebert/modeling.py

```diff
@@ -0,0 +1,691 @@
+#
+#     def __init__(self, squeezebert):
+#         super().__init__()
+#         self.squeezebert = squeezebert
+#         self.initializer_range = self.squeezebert.config['initializer_range']
+#         self.cls = SqueezeBertPreTrainingHeads(create_config(self.squeezebert.config))
+#         self.init_weights()
+#
+#     def forward(self,
+#                 input_ids,
+#                 token_type_ids=None,
+#                 position_ids=None,
+#                 attention_mask=None,
+#                 masked_positions=None):
+#         with paddle.static.amp.fp16_guard():
+#             outputs = self.bert(
+#                 input_ids,
+#                 token_type_ids=token_type_ids,
+#                 position_ids=position_ids,
+#                 attention_mask=attention_mask)
+#             sequence_output, pooled_output = outputs[:2]
+#             prediction_scores, seq_relationship_score = self.cls(
+#                 sequence_output, pooled_output, masked_positions)
+#             return prediction_scores, seq_relationship_score
+
+
+class SqueezeBertForSequenceClassification(SqueezeBertPreTrainedModel):
+
+    def __init__(self, squeezebert, num_classes=2, dropout=None):
+        super().__init__()
+        self.num_classes = num_classes
+        self.squeezebert = squeezebert
+        self.dropout = nn.Dropout(dropout if dropout is not None else
+                                  self.squeezebert.config["hidden_dropout_prob"])
+        self.classifier = nn.Linear(self.squeezebert.config["hidden_size"],
+                                    num_classes)
+        self.apply(self.init_weights)
+
+    def forward(self,
+                input_ids,
+                token_type_ids=None,
+                position_ids=None,
+                attention_mask=None):
+        _, pooled_output = self.squeezebert(
+            input_ids,
+            token_type_ids=token_type_ids,
+            position_ids=position_ids,
+            attention_mask=attention_mask)
+
+        pooled_output = self.dropout(pooled_output)
+        logits = self.classifier(pooled_output)
+        return logits
+
+
+class SqueezeBertForQuestionAnswering(SqueezeBertPreTrainedModel):
+
+    def __init__(self, squeezebert, dropout=None):
+        super().__init__()
+        self.squeezebert = squeezebert
+        self.classifier = nn.Linear(self.squeezebert.config["hidden_size"], 2)
+        self.apply(self.init_weights)
+
+    def forward(self, input_ids, token_type_ids=None):
+        sequence_output, _ = self.squeezebert(
+            input_ids,
+            token_type_ids=token_type_ids,
+            position_ids=None,
+            attention_mask=None)
+        logits = self.classifier(sequence_output)
+        logits = paddle.transpose(logits, perm=[2, 0, 1])
+        start_logits, end_logits = paddle.unstack(x=logits, axis=0)
+        return start_logits, end_logits
+
+
+class SqueezeBertForTokenClassification(SqueezeBertPreTrainedModel):
+

```

Comment:
Need more doc for models for specific tasks

Reply:
PLZ review latest submit.

Reply:
> Need more doc for models for specific tasks

I have not got your message for two days, is there any wrong?